src/fix_hmc.cpp

```diff
@@ -597,8 +613,7 @@ void FixHMC::end_of_step()
   nattempts++;
 
   // Compute proposed unwrapped positions and displacements:
-  if (rigid_flag)
-    atom_positions(xu); 
+  if (rigid_flag) atom_positions(xu);
```

Comment:
Why do they sometimes move these onto one line and sometimes move them onto two lines smh. I like this one line tho I'm just saying.